performance-testing/performance-testing-core/src/main/java/uk/gov/gchq/gaffer/performancetesting/query/QueryTest.java

```diff
@@ -114,7 +113,7 @@ private void queryBatch(final Supplier<? extends ElementSeed> elementSeedSupplie
         final long startTime = System.currentTimeMillis();
         final long numResults;
         try {
-            final CloseableIterable<? extends Element> results = graph.execute(getElements, new User());
+            final Iterable<? extends Element> results = graph.execute(getElements, new User());
```

Comment:
Does this need to be closed in a `finally` block? I know it wasn't before, just will be a memory leak if there is an exception (I think).